ghe-check-hotpatch.sh

```diff
@@ -89,11 +97,24 @@ check_log () {
 verify_running_image_tags () {
 # Get the running, and expected, hashes for all containerized service
 # and verify if they match.
+
+  # Cache JSON so we don't call sudo so many times
+  expected_json="$(sudo cat /data/user/docker/image/overlay2/repositories.json)"
```

Comment:
Each loop, a shell was being spun up invoking an instance of `sudo`. This will fill the logs unnecessarily and slow things down.